xendit-android/src/main/java/com/xendit/Models/Authentication.java

```diff
@@ -23,10 +23,14 @@
     @SerializedName("payer_authentication_url")
     private String payerAuthenticationUrl;
 
+    @SerializedName("masked_card_number")
+    private String maskedCardNumber;
```

Comment:
I think we're not returning this on authentication right?

Reply:
oh, token get it from authentication ya.. I forgot, will this work on `createAuthentication`?

Reply:
yes this works on authentication only